src/test/scala/it/pagopa/interop/attributeregistryprocess/utils/FakeDependencies.scala

```diff
@@ -0,0 +1,130 @@
+      contexts: Seq[(String, String)]
+    ): Future[Tenant] = Future.successful(fakeTenant)
+
+    override def addTenantAttribute(tenantId: UUID, seed: TenantAttribute)(implicit
+      contexts: Seq[(String, String)]
+    ): Future[Tenant] = Future.successful(fakeTenant)
+
+    override def getTenant(tenantId: UUID)(implicit contexts: Seq[(String, String)]): Future[Tenant] =
+      Future.successful(fakeTenant)
+
+    override def getTenantByExternalId(externalId: ExternalId)(implicit
+      contexts: Seq[(String, String)]
+    ): Future[Tenant] = Future.successful(fakeTenant)
+
+    override def getTenantAttribute(tenantId: UUID, attributeId: UUID)(implicit
+      contexts: Seq[(String, String)]
+    ): Future[TenantAttribute] = Future.successful(fakeAttribute)
+  }
+
+  case class FakeAgreementProcess() extends AgreementProcessService {
+    override def computeAgreementsByAttribute(consumerId: UUID, attributeId: UUID)(implicit
+      contexts: Seq[(String, String)]
+    ): Future[Unit] = Future.unit
+  }
+
+  case class FakeAgreementManagement() extends AgreementManagementService {
+    override def getAgreements(producerId: UUID, consumerId: UUID, states: Seq[AgreementState])(implicit
+      contexts: Seq[(String, String)]
+    ): Future[Seq[Agreement]] = Future.successful(Seq(agreement))
+  }
+
+  case class FakeCatalogManagement() extends CatalogManagementService {
+    override def getEServiceById(eServiceId: UUID)(implicit contexts: Seq[(String, String)]): Future[EService] =
+      Future.successful(eService)
+  }
+
+  class FakeReadModelService extends ReadModelService {
+    override def findOne[T](collectionName: String, filter: Bson)(implicit
+      evidence$1: JsonReader[T],
+      ec: ExecutionContext
+    ): Future[Option[T]] = Future.successful(None)
+    override def find[T](collectionName: String, filter: Bson, offset: Int, limit: Int)(implicit
+      evidence$2: JsonReader[T],
+      ec: ExecutionContext
+    ): Future[Seq[T]] = Future.successful(Nil)
+    override def find[T](collectionName: String, filter: Bson, projection: Bson, offset: Int, limit: Int)(implicit
+      evidence$3: JsonReader[T],
+      ec: ExecutionContext
+    ): Future[Seq[T]] = Future.successful(Nil)
+    override def aggregate[T](collectionName: String, pipeline: Seq[Bson], offset: Int, limit: Int)(implicit
+      evidence$4: JsonReader[T],
+      ec: ExecutionContext
+    ): Future[Seq[T]] = Future.successful(Nil)
+
+    override def close(): Unit = ()
+  }
+
+  val fakeTenant: Tenant = Tenant(
+    id = UUID.randomUUID(),
+    selfcareId = None,
+    externalId = ExternalId("IPA", "something"),
+    features = Seq(TenantFeature(certifier = Some(Certifier("SOMETHING")))),
+    attributes = Nil,
+    createdAt = OffsetDateTime.now(),
+    updatedAt = None,
+    mails = Nil,
+    name = "test_name"
+  )
+
+  val fakeAttribute: TenantAttribute = TenantAttribute(None, None, None)
```

Comment:
[same](https://github.com/pagopa/interop-be-attribute-registry-process/pull/1/files#r1135229414)